tests/nested/manual/refresh-from-old-base/task.yaml

```diff
@@ -0,0 +1,67 @@
+summary: Test refreshing from old snapd and base prior to required symlink
+
+details: |
+    Base snaps used to not initialize /snap/snapd/current symlink to seed.
+    However the question is what happen when starting with an old image,
+    and updating snapd before the base. This makes sure that this initial
+    symlink is not involved with updates.
+
+systems:
+  - ubuntu-22.04-64
+
+environment:
+  NESTED_BUILD_SNAPD_FROM_CURRENT: false
+  NESTED_REPACK_BASE_SNAP: false
+  NESTED_FORCE_SNAKEOIL_KEYS: true
+
+prepare: |
+    snap download --channel="latest/${BASE_CHANNEL}" core22 --basename=core22-new
+
+    # shellcheck source=tests/lib/prepare.sh
+    . "$TESTSLIB"/prepare.sh
+    [ -d new-snap/ ] || mkdir -p new-snap/
+    build_snapd_snap new-snap/
+    for f in new-snap/snapd_*.snap; do
+        mv "${f}" snapd-new.snap
+    done
+    rm -rf new-snap
+
+    snap download --revision=1033 core22 --basename=core22-old
+    unsquashfs -d core22-old core22-old.snap
+    rm core22-old.snap
+    snap pack core22-old/ "$(tests.nested get extra-snaps-path)"
```

Comment:
Why do we download, unpack and repack the base snap without changes?

Reply:
I have added a comment explaining the reason. Which is to not let auto-refresh kick in.